### ISSUE TITLE
Installation Error CyanogenMod

### ISSUE THREAD

**Issue by sahne123** [dmfs/opentasks#218]
Hi,

I am using F-Droid and CyanogenMod (Android 5.1.1). The last update to Version 1.1.8.2 failed with "Installation Error" (was no real update a message said i have to reinstall the app). Since then i tried to reinstall manually. I also tried to download the apk file directly from github - same error. I have also tried to update Cyanogenmod. It is not even possible to install an older version of opentasks, wich runs perfectly before.

I'm afraid i cant deliver more information. Is there a way to gather more information during an installaton process? tried catlog.

**Comment by dmfs** [comment]
Which sync app did you install? I would expect that there is at least some kind of information in the system log :-|
Can you try to install the app with `adb` from a PC or Mac?

**Comment by sahne123** [comment]
I am using CalDAV-Sync. It was possible to install Version 1.1.6 via the amazon app store. At least something.

I can try to install with adb and will report.

**Comment by dmfs** [comment]
Did you install CalDAV-Sync from Amazon as well? We sometimes hear that installing the apps from different sources doesn't seem to work. I'll update the app on Amazon as well.

**Comment by sahne123** [comment]
Yes, CalDAV-Sync is from amazon. It can also be the that the initial installation from tasks was from amazon as well. Im not sure about that. 

F-Droid seems to offer the update whether or not the app was installed by F-Droid.

**Comment by dmfs** [comment]
I've updated OpenTasks on Amazon, should be available soon.
